certified-connectors/Witivio/apiDefinition.swagger.json

```diff
@@ -1194,5 +1262,19 @@
       "oauth2_auth": []
     }
   ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://witivio.com"
```

Comment:
@tdechanterac Please add domain to your website, like below:
`"https://www.witivio.com"`